project_5.html

```diff
@@ -0,0 +1,721 @@
+				<li class="list-group-item">
+					<p>
+						One of the player’s buttons must toggle the music between “Play” mode and “Pause” mode.
+						<ul>
+							<li>In “Play” mode, the player must playback a song. It should resume the song that was previously paused if the “Skip/Next Song” button was not pressed.</li>
+							<li>In “Pause” mode, no song should be played.</li>
+						</ul>
+					</p>
+				</li>
+				<li class="list-group-item">
+					<p>
+						The second button should be the “Skip/Next Song” button, which interrupts the current song and plays a new song.
+						<ul>
+							<li>Each press of this button must cycle through all songs loaded on the MicroSD card.</li>
+							<li>If the player was in “Pause” mode when the “Skip/Next Song” button is pressed, the player must immediately enter “Play” mode.</li>
+						</ul>
+					</p>
+				</li>
+				<li class="list-group-item">The speaker must be silent when no song is played. No static or white noise.</li>
+				<li class="list-group-item">The speaker must clearly output music when a song is played.</li>
+				<li class="list-group-item">The circuit must be built on a breadboard.</li>
+			</ul>
+
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-parts">Parts</h2>
+			<!-- ======= Part Table ======= -->
+			<table class="table pj-part-table mb-5">
+				<thead>
+					<tr>
+						<th scope="col">Part Name</th>
+						<th scope="col">Qty</th>
+					</tr>
+				</thead>
+				<tbody>
+					<tr>
+						<td>
+							Jumper Wire
+						</td>
+						<td>?</td>
+					</tr>
+					<tr>
+						<td>Breadboard</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part pj-part-link"
+								data-bs-content="<img src='./assets/images/parts/ops_part_arduino_nano.png' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0"
+								href="./assets/datasheets/ops_datasheet_arduino_nano.pdf" target="_blank"
+								rel="noopener noreferrer">
+								Arduino Nano
+							</a>
+						</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/project_5/ops_project_5_microsd.jpg' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								MicroSD Card
+							</a>
+						</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							Pushbutton Switch
+						</td>
+						<td>2</td>
+					</tr>
+					<tr>
+						<td>
+							Resistor, 1kΩ
+						</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							Resistor, 10kΩ
+						</td>
+						<td>1</td>
+					</tr>
+					<tr>
+						<td>
+							<a class="pj-part"
+								data-bs-content="<img src='./assets/images/project_5/ops_project_5_mini_mp3.webp' height='150px' width='auto'>"
+								data-bs-trigger="hover focus" data-bs-toggle="popover" tabindex="0">
+								DFPlayer Mini Module
```

Comment:
Please make this a link to the datasheet (find it on Drive in the project 5 folder)